docs/reST/ref/event.rst

```diff
@@ -151,6 +152,8 @@ attributes.
    If you are only taking specific events from the queue, be aware that the
    queue could eventually fill up with the events you are not interested.
 
+   If ``pump`` is ``True``, then :func:`pygame.event.pump()` will be called.
```

Comment:
For all the API changes, you might want to consider adding a version notification (e.g. `.. versionadded:: 1.9.5`) with a short description of what is new.

Reply:
Sure. I think the other documentation should be updated to use the directive instead of text also? If the text for ".. versionadded X" is changed to "New in pygame X".